internal/pkg/externalplugins/owners/owners.go

```diff
@@ -168,19 +168,41 @@ func (s *Server) ListOwners(org string, repo string, number int,
 		return nil, err
 	}
 
+	branchName := pull.Base.Ref
+	branchConfig := opts.Branches[branchName]
+
 	// When we cannot find the require label from the PR, try to use the default require lgtm.
 	if requireLgtm == 0 {
-		requireLgtm = opts.DefaultRequireLgtm
+		if branchConfig.DefaultRequireLgtm != 0 {
+			requireLgtm = branchConfig.DefaultRequireLgtm
+		} else {
+			requireLgtm = opts.DefaultRequireLgtm
+		}
+	}
+
+	// Notice: If the branch of the PR has extra trust team config, it will override the repository config.
+	var trustTeams []string
+
+	if len(branchConfig.TrustedTeams) != 0 {
```

Comment:
I don't know if the array in yaml will be nil or not, could you look it up and decide if you want to determine nil ?

Reply:
It seems that even if the len() function is executed on a nil slice, it will return 0.

![Snipaste_2020-12-16_17-17-41](https://user-images.githubusercontent.com/5086433/102328911-a89d0480-3fc2-11eb-97a6-4f0c08ae1b27.png)

Reply:
I think it is better to use nil, what do you think?